contracts/test/SampleERC20.sol

```diff
@@ -1,34 +1,28 @@
-// SPDX-License-Identifier: UNLICENSED
-pragma solidity =0.8.16;
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.18;
```

Comment:
should we keep the version fixed like =0.8.18? or is there a benefit of using the ^ instead? would there be anycase of newer version breaking backward compatibility